lib/ohai/plugins/solaris2/network.rb

```diff
@@ -119,7 +125,7 @@ def arpname_to_ifname(iface, arpname)
 
     so = shell_out("arp -an")
     so.stdout.lines do |line|
-      if line =~ /([0-9a-zA-Z]+)\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\w+)\s+([a-zA-Z0-9\.\:\-]+)/
+      if line =~ /([0-9a-zA-Z]+)\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\s+(\w+)?\s+([a-zA-Z0-9\.\:\-]+)/
```

Comment:
If you have an example of what this line matches handy, it'd be :100: to have a comment about it. Long regex is long.